science-and-technology-for-the-built-environment.csl

```diff
@@ -0,0 +1,549 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```

Comment:
@mitchpaulus, if this style is only used in a single language, it should have a `default-locale` attribute, see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale